<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>useritem</class>
 <widget class="QWidget" name="useritem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>295</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>295</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>黄忆杭</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_feeling">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>今天吃了条鱼 叫艾泥芝士不鱼</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
